docs/Ecosystem.md

```diff
@@ -29,6 +29,7 @@ Plugins maintainted by the fastify team are listed under [Core](#core) while plu
 
 #### [Community](#community)
 - [`fastify-apollo`](https://github.com/coopnd/fastify-apollo) Run an [Apollo Server](https://github.com/apollographql/apollo-server) to serve GraphQL with Fastify.
+- [`fastify-firebase-auth`](https://github.com/oxsav/fastify-firebase-auth) Firebase Authentication for Fastify - Support all the methods relate to the authentication API.
```

Comment:
How about "Firebase authentication for Fastify supporting all of the methods relating to the authentication API."?

Reply:
Seems better :) . Modified it already. Don't know if really want to merge it regarding this thread: https://github.com/oxsav/fastify-firebase-auth/issues/1 

Let me know what do you think

Reply:
I think we can work on this plugin and once is ready we can merge this pr.
What do you think? :)

Reply:
Sorry for late response. Yep I agree! thanks for the feedback ;)